src/styles/main.scss

```diff
@@ -0,0 +1,42 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
+  padding-inline: $cellPading;
+
+  &__day {
+    width: $cellSize;
+    height: $cellSize;
+    background-color: $cellCollor;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    border: solid 1px black;
+
+    font-family: $cellFontFamily;
+    font-size: $cellFontHeight;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY($translateY);
+      transition: transform $transformDuration;
```

Comment:
The transition property should be defined within the element itself, not on the hover state. This is to ensure that the transition effect applies both when the element is hovered over and when the hover state is removed.